js/main.js

```diff
@@ -200,7 +200,6 @@ function showMenu() {
 
 function hideMenu() {
   menu.style.display = "none"; // Hide the menu
-  document.querySelector("#chat-input textarea").focus();
```

Comment:
Can you make this not apply on mobile but apply on desktop? That focus is useful when using a physical keyboard.